src/transaction/declare_v2.rs

```diff
@@ -350,26 +316,22 @@ impl DeclareV2 {
         let mut tx_execution_context =
             self.get_execution_context(block_context.validate_max_n_steps);
 
-        let call_info = if self.skip_execute {
-            None
+        if self.skip_execute {
```

Comment:
is this an inteded behavior? because it will cause the simulation to fail instead of calculating something.

Reply:
you're right! reverted

Reply:
this seems like a refactor, previously `verify_no_calls_to_other_contracts()` would throw the same error if `call_info` were `None`

Reply:
yes, that's the case . It is just a refactor of what was previously